src/app/[locale]/_components/LandingPageHeader.tsx

```diff
@@ -51,14 +50,6 @@ export default function LandingPageHeader({ locale }: { locale: string }) {
                 alt={"Project Protocol logo"}
               />
             </div>
-            <span
-              className={
-                "w-100 d-md-inline fw-medium pe-auto text-white " +
-                styles.navbarTitle
-              }
-            >
-              Project Protocol
-            </span>
           </NavbarBrand>
           <LocaleSwitcher locale={locale} dark />
```

Comment:
Best for alignment and hierarchy if we move the link here. Also I think it's better to use `<a />` tags and make them look like buttons rather than buttons that act as links.
```suggestion
          <div className="d-flex flex-row align-items-center gap-1">
            <LocaleSwitcher locale={locale} dark />
            <a
              className="btn btn-primary btn-sm"
              href="https://youngwomenfree.app.neoncrm.com/forms/project-protocol"
              target="_blank"
            >
              {t("donate")}
            </a>
          </div>
```